UPGRADING

```diff
@@ -105,6 +105,14 @@ PHP 8.3 UPGRADE NOTES
 10. New Global Constants
 ========================================
 
+- Curl:
+  . CURLINFO_CAPATH
+  . CURLINFO_CAINFO
+  . CURLOPT_SSH_HOSTKEYFUNCTION
+  . CURLOPT_PROTOCOLS_STR
+  . CURLOPT_REDIR_PROTOCOLS_STR
+  . CURLOPT_QUICK_EXIT
+
```

Comment:
I have the feeling you might need to update `interface.c` too, what do you think ?

Reply:
Yep you're right, thanks for noticing, I'll get to updating it soon!